src/lvl_script.c

```diff
@@ -6564,6 +6633,8 @@ const struct CommandDesc command_desc[] = {
   {"CREATE_EFFECTS_LINE",               "LLNNNN  ", Cmd_CREATE_EFFECTS_LINE, &create_effects_line_check, &create_effects_line_process},
   {"IF_SLAB_OWNER",                     "NNP     ", Cmd_IF_SLAB_OWNER, NULL, NULL},
   {"IF_SLAB_TYPE",                      "NNS     ", Cmd_IF_SLAB_TYPE, NULL, NULL},
+  {"PRINT",                             "NAA     ", Cmd_PRINTFX, NULL, NULL},
```

Comment:
> 
> 
> instead of adding the Cmd_PRINTFX and Cmd_MESSAGEFX, why not simply replace the existing Cmd_PRINT and Cmd_MESSAGE?

Well, I highly doubt the original commands did what my ones do. But this got me thinking: if we ever find out and replicate what the originals did, do we need to rename my ones?

Reply:
I've looked into this further, and yeah, you could get rid of Cmd_MESSAGEFX.

What's really confusing now is if you use the message command without level_version(1) you get a log message that the command only works in beta. Checking at the other commands, they just query for the level version:

This is for Cmd_RANDOM:
```
                if ((funcmd_desc->index != Cmd_RANDOM) && (level_file_version == 0)) {
                    SCRPTERRLOG("The function \"%s\" used within command \"%s\" is not supported in old level format", funcmd_desc->textptr, scline->tcmnd);
                    break;
```
I would like you to make it like that. So, always have 'MESSAGE' go to Cmd_MESSAGE, and have Cmd_MESSAGE handle the new command with level_version(1) and give back a log message mentioning the old level format when you don't have (1) enabled.

Reply:
> 
> 
> I've looked into this further, and yeah, you could get rid of Cmd_MESSAGEFX.
> 
> What's really confusing now is if you use the message command without level_version(1) you get a log message that the command only works in beta. Checking at the other commands, they just query for the level version:
> 
> This is for Cmd_RANDOM:
> 
> ```
>                 if ((funcmd_desc->index != Cmd_RANDOM) && (level_file_version == 0)) {
>                     SCRPTERRLOG("The function \"%s\" used within command \"%s\" is not supported in old level format", funcmd_desc->textptr, scline->tcmnd);
>                     break;
> ```
> 
> I would like you to make it like that. So, always have 'MESSAGE' go to Cmd_MESSAGE, and have Cmd_MESSAGE handle the new command with level_version(1) and give back a log message mentioning the old level format when you don't have (1) enabled.

I don't think that'll work; my MESSAGE command has two parameters, whereas the original only had one.

Reply:
The other commands handle that just fine too. Look at command_creature_available for example.

Reply:
I'm not seeing anything in command_creature_available about multiple versions with different parameters.